src/app/pages/Collectibles/CollectiblesTab/index.tsx

```diff
@@ -0,0 +1,171 @@
+  const { isSyncing: tokensAreSyncing } = useSyncTokens();
+  const metadatasLoading = useTokensMetadataLoadingSelector();
+
+  const [areDetailsShown, setDetailsShown] = useLocalStorage(LOCAL_STORAGE_TOGGLE_KEY, false);
+
+  const toggleDetailsShown = useCallback(() => void setDetailsShown(val => !val), [setDetailsShown]);
+
+  const { data: collectibles = [], isValidating: readingCollectibles } = useCollectibleTokens(
+    chainId,
+    publicKeyHash,
+    true
+  );
+
+  const collectiblesSlugs = useMemoWithCompare(
+    () => collectibles.map(collectible => collectible.tokenSlug).sort(),
+    [collectibles],
+    isEqual
+  );
+
+  const { filteredAssets, searchValue, setSearchValue } = useFilteredAssetsSlugs(collectiblesSlugs, false);
+
+  const shouldScrollToTheTabsBar = collectibles.length > 0;
+  useEffect(() => void scrollToTheTabsBar(), [shouldScrollToTheTabsBar, scrollToTheTabsBar]);
+
+  const isSyncing = tokensAreSyncing || metadatasLoading || readingCollectibles;
+
+  return (
+    <div className="w-full max-w-sm mx-auto">
+      <div className={clsx('my-3', popup && 'mx-4')}>
+        <div className="mb-4 w-full flex items-strech">
+          <SearchAssetField
+            value={searchValue}
+            onValueChange={setSearchValue}
+            containerClassName="mr-2"
+            testID={AssetsSelectors.searchAssetsInputCollectibles}
+          />
+
+          <Popper
+            placement="bottom-end"
+            strategy="fixed"
+            popup={props => (
+              <ManageButtonDropdown
+                {...props}
+                areDetailsShown={areDetailsShown}
+                toggleDetailsShown={toggleDetailsShown}
+              />
+            )}
+          >
+            {({ ref, opened, toggleOpened }) => (
+              <ButtonForManageDropdown
+                ref={ref}
+                opened={opened}
+                tooltip={t('manageAssetsList')}
+                onClick={toggleOpened}
+                testID={AssetsSelectors.manageButton}
+                testIDProperties={{ listOf: 'Collectibles' }}
+              />
+            )}
+          </Popper>
+        </div>
+
+        {filteredAssets.length === 0 ? (
+          isSyncing ? (
+            <SyncSpinner className="mt-6" />
+          ) : (
+            <div className="w-full border rounded border-gray-200">
+              <p className={'text-gray-600 text-center text-xs py-6'}>
+                <T id="zeroCollectibleText" />
+              </p>
+            </div>
+          )
+        ) : (
+          <>
+            <div className="grid grid-cols-3 gap-1">
+              {filteredAssets.map(slug => (
+                <CollectibleItem
+                  key={slug}
+                  assetSlug={slug}
+                  accountPkh={publicKeyHash}
+                  areDetailsShown={areDetailsShown}
+                />
+              ))}
+            </div>
+
+            {isSyncing && <SyncSpinner className="mt-6" />}
```

Comment:
looks like duplicated.
If `filteredAssets.length === 0 && isSyncing` two SyncSpinner should be rendered?